multidim-interop/src/generator.ts

```diff
@@ -79,14 +86,27 @@ export async function buildTestSpecs(versions: Array<Version>): Promise<Array<Co
             muxer: test.muxer,
             security: test.sec,
         })
-    )).concat(quicQueryResults.concat(quicV1QueryResults).concat(webtransportQueryResults).map((test): ComposeSpecification => buildSpec(containerImages, {
-        name: `${test.id1} x ${test.id2} (${test.transport})`,
-        dialerID: test.id1,
-        listenerID: test.id2,
-        transport: test.transport,
-        muxer: "quic",
-        security: "quic",
-    })))
+    )).concat(
+        quicQueryResults
+        .concat(quicV1QueryResults)
+        .concat(webtransportQueryResults)
+        .map((test): ComposeSpecification => buildSpec(containerImages, {
+            name: `${test.id1} x ${test.id2} (${test.transport})`,
+            dialerID: test.id1,
+            listenerID: test.id2,
+            transport: test.transport,
+            muxer: "quic",
+            security: "tls",
```

Comment:
@MarcoPolo what do you think of this change? It doesn't break Go `quic` tests, and allows Rust to have one less Variant in the enum. Plus I'd say it makes sense semantically.

Reply:
I don’t think it makes sense. In every other context tls means add a tls encryption layer on top of the transport.

I know quic uses tls underneath and that isn’t a hardcoded decision (some different encryption layer could be possible in the future). But for now I think we should keep tls meaning add a tls security layer to the transport.

Reply:
yeah you are right Marco, I reverted it int this PR so we can merge and address the `rust-libp2p` interop. But I also agree with @thomaseizinger on https://github.com/libp2p/test-plans/pull/100#discussion_r1071735120 if you agree we can submit a PR not using muxer nor transport for `quic` and `webrtc` cases